test/e2e_node/system_node_critical_test.go

```diff
@@ -38,6 +38,14 @@ var _ = SIGDescribe("SystemNodeCriticalPod [Slow] [Serial] [Disruptive] [NodeFea
 	// this test only manipulates pods in kube-system
 	f.SkipNamespaceCreation = true
 
+	ginkgo.AfterEach(func() {
+		if framework.TestContext.PrepullImages {
+			// The test may cause the prepulled images to be evicted,
```

Comment:
When do you say about evicting images, does it mean the kubelet will remove them?

Reply:
When the node is under `DiskPressure`, it will do a lot of stuff, including deleting unused images. (Eviction is maybe the wrong word tho.). Since all e2e_node tests use the `Never` pull policy, this causes problems.

More info here: https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/